docs/configure/integration.md

```diff
@@ -0,0 +1,315 @@
+
+#### Default configuration
+
+The base Typescript configuration uses [`babel-loader`](https://webpack.js.org/loaders/babel-loader/) for Typescript transpilation, and optionally [`fork-ts-checker-webpack-plugin`](https://github.com/TypeStrong/fork-ts-checker-webpack-plugin) for checking.
+
+Each framework uses the base configuration unless otherwise specified:
+
+- Angular ignores the base and uses `ts-loader` and `ngx-template-loader`.
+- Vue ignores the uses `ts-loader` and applies it to both `.tsx` and `.vue` files.
+- React adds `react-docgen-typescript-plugin` to the base.
+
+
+#### Main.js configuration
+
+To make it easier to configure Typescript handling, use the `typescript` field in your [`.storybook/main.js`](./overview#configure-story-rendering).
+
+The following code snippets shows the fields for you to use with TypeScript:
+
+```js
+// .storybook/main.js
+module.exports = {
+  typescript: {
+    check: false,
+    checkOptions: {},
+    reactDocgen: 'react-docgen-typescript',
+    reactDocgenTypescriptOptions: {
+      shouldExtractLiteralValuesFromEnum: true,
+      propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
+    },
+  },
+};
+```
+
+<div style="background-color:#F8FAFC">
+TODO: vet SEE DOCS link
+</div>
+
+|Field	                         |Framework	    |Description	                                                                          |Type      |
+|:-------------------------------|:------------:|:---------------------------------------------------------------------------------------:|:--------:|
+|**check**                       |All	        |optionally run fork-ts-checker-webpack-plugin	                                          |boolean   |
+|**checkOptions**	             |All	        |Options to pass to fork-ts-checker-webpack-plugin if it's enabled	                      |See docs  |
+|**reactDocgen**	             |React	        |which variant docgen processor to run	`'react-docgen-typescript'                        |N/A       |
+|**reactDocgenTypescriptOptions**|React	        |Options to pass to react-docgen-typescript-plugin if react-docgen-typescript is enabled. |See docs  |
+
+
+
+### Styling and CSS
+
+There are many ways to include CSS in a web application, and correspondingly there are many ways to include CSS in Storybook. Usually it is best to try and replicate what your application does with styling in Storybook’s configuration.
+
+#### CSS-in-JS
+
+CSS-in-JS libraries are designed to use basic JavaScript. They often work in Storybook without any extra configuration. Some libraries expect components to be rendered in a specific rendering “context” (such as to provide themes) and you may need to add a [global decorator](../writing-stories/decorators#global-decorators) to supply it.
+
+#### Importing CSS files
+
+<div style="">
+TODO: check note on sb-storybook-6_0_more_feedback.txt for this
```

Comment:
Where is this note?

Reply:
that note slipped through. It was refering to my notes file based on the feedback in the get started section. It's now removed and marking it as solved.